git-changelist-maven-extension/src/main/java/io/jenkins/tools/incrementals/git_changelist_maven_extension/Main.java

```diff
@@ -100,6 +100,7 @@ public void afterSessionStart(MavenSession session) throws MavenExecutionExcepti
                     hash = head.abbreviate(ABBREV_LENGTH).name();
                     try (RevWalk walk = new RevWalk(repo)) {
                         RevCommit headC = walk.parseCommit(head);
+                        timestamp = DateTimeFormatter.ISO_INSTANT.format(Instant.ofEpochSecond(headC.getCommitTime()));
```

Comment:
Should probably a string such as `2024-04-12T19:40:42Z` which seems to align with https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=74682318#Reproducible/VerifiableBuilds-OutputArchiveEntriesTimestamp though it is vague about allowed precision.